labordata/lm10
# Fix attribute errors when scraping and filing.json structure

## Overview

Note: The issue here is very similar to the one resolved in [this lm20 pr](https://github.com/labordata/lm20/pull/13). So, much of this solution has been co-opted from that.

The `content-dispositions` when looking for filenames as of late have been either of type `str` or `None`. So the `decode()` method that has been in use has been raising `AttributeErrors`. This branch accounts for the `content-disposition` being a either of those values, and passes the filename found within.

After resolving that error, `jq` had issues parsing out the `files` attributes of objects in `filing.json` since they were nested objects. This branch also flattens out that object with a make command to get the command working.

- Closes #5 

## Testing Instructions

 * Run `make lm20.db` or look at the checks performed on this PR
 * Confirm that everything in the "build database" portion runs without the str/None attribute error
 * Confirm that the data in the db is what we expect
